src/index.js

```diff
@@ -81,6 +81,7 @@ function main(client, options, _performance) {
 	};
 
 	client.getActiveKeys('web').listen(function (keys) {
+		functions.clear();
```

Comment:
Why does "functions" live outside this scope? Should it not just be constructed each time from the newly activated keys and passed to invokeFunctions?

Reply:
👍